build.gradle

```diff
@@ -848,6 +848,11 @@ project('test:system') {
 
     task startK8SystemTests(type: Test) {
         tasks.withType(Exec) { environment "tier2Type", System.getProperty("tier2Type", "nfs") }
+        tasks.withType(Exec) { environment "imageVersionPrOp",System.getProperty("imageVersionPrOp", "latest") }
+        tasks.withType(Exec) { environment "imageVersionBkOp",System.getProperty("imageVersionBkOp", "latest") }
+        tasks.withType(Exec) { environment "imageVersionZkOp",System.getProperty("imageVersionZkOp", "latest") }
+        tasks.withType(Exec) { environment "chartName",System.getProperty("chartName", "pravega") }
+        tasks.withType(Exec) { environment "repository",System.getProperty("repository", "pravega") }
```

Comment:
Could you please provide more details about this property? Also, could we rename it so that it indicates to the user that it is only used by a helm command?

Reply:
added description:         // The below properties are used to to specify the pravega published chart and repository to deploy pravega,bookkeeper & zookkeeper operators using helm docker image properties